src/components/SelectionList/BaseListItem.tsx

```diff
@@ -23,6 +24,7 @@ function BaseListItem<TItem extends ListItem>({
     onCheckboxPress,
     onDismissError = () => {},
     rightHandSideComponent,
+    checkmarkDirection = 'left',
```

Comment:
Wouldn't position be better in this case?
```suggestion
    checkmarkPosition = 'left',
```